package protoFile.hall.ModifyRoomInfo1822;
message ModifyRoomInfoRequest{
      required int32 id=1;//角色id
      optional string groupName=2;//队伍名称
      optional string groupPassword=3;//密码
      optional int32 copySceneId=4;//副本id
      optional int32 copyLevel=5;//难道级别0普通1精英2英雄
} 
message ModifyRoomInfoResponse{
      required bool result = 1;
      optional string message = 2;
}